app/views/clients/index.html.slim

```diff
@@ -1,4 +1,4 @@
-= will_paginate
+= will_paginate @clients, renderer: BootstrapPagination::Rails
```

Comment:
I don't know. I was just following (C&P) the instructions on [bootstrap-ruby/will_paginate-bootstrap](https://github.com/bootstrap-ruby/will_paginate-bootstrap)
Could be that this could also be skipped.